deploy/cdkpipeline/cdkpipeline.py

```diff
@@ -0,0 +1,147 @@
+# importing modules
+from aws_cdk import (
+    aws_ssm as ssm,
+    pipelines,
+    core as cdk,
+    aws_codepipeline as codepipeline,
+    aws_s3 as s3,
+    aws_codepipeline_actions as codepipeline_actions,
+    aws_iam as iam
+)
+from stacks.sscheck import SampleSheetCheckFrontEndStack
+
+class SampleSheetCheckStage(cdk.Stage):
+    def __init__(self, scope: cdk.Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        # Create stack defined on stacks folder
+        SampleSheetCheckFrontEndStack(self, "SampleSheetCheckFrontEnd")
+
+# Class for the CDK pipeline stack
+class CdkPipelineStack(cdk.Stack):
+
+    def __init__(self, scope: cdk.Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        # Load SSM parameter for GitHub repo (Created via Console)
+        codestar_arn = ssm.StringParameter.from_string_parameter_attributes(self, "codestarArn",
+            parameter_name="/sscheck/codestar-connection-arn"
+        ).string_value
+
+        cloud_artifact = codepipeline.Artifact(
+            artifact_name="sscheck_pipeline_cloud"
+        )
+
+        source_artifact = codepipeline.Artifact(
+            artifact_name="sscheck_pipeline_source",
+        )
+
+        # Fetch github repository for changes
+        code_star_action = codepipeline_actions.CodeStarConnectionsSourceAction(
+            connection_arn = codestar_arn,
+            output = source_artifact,
+            owner = "umccr",
+            repo = "samplesheet-check-frontend",
+            branch = "implement-pipeline",
```

Comment:
As discussed, any merges to `dev` branch deploy to DEV account.  

- At repo setting, create branch `dev` and set it to default. 
- Keep the `main` branch in sync with `dev`. i.e forward only merges for linear history `git checkout main && git merge --ff-only dev`
- And any merges to `main` branch should trigger deploy to PROD account.